editor/import/editor_scene_importer_gltf.cpp

```diff
@@ -2079,7 +2079,7 @@ void EditorSceneImporterGLTF::_import_animation(GLTFState &state, AnimationPlaye
 				animation->track_set_path(track_idx, node_path);
 
 				if (track.weight_tracks[i].interpolation <= GLTFAnimation::INTERP_STEP) {
-					animation->track_set_interpolation_type(track_idx, track.weight_tracks[i].interpolation == GLTFAnimation::INTERP_STEP ? Animation::INTERPOLATION_NEAREST : Animation::INTERPOLATION_NEAREST);
+					animation->track_set_interpolation_type(track_idx, Animation::INTERPOLATION_NEAREST);
```

Comment:
The original code was visibly a typo, one of the two was probably meant to be `Animation::INTERPOLATION_LINEAR`. CC @reduz